connexion/exceptions.py

```diff
@@ -83,8 +83,7 @@ class OAuthProblem(Unauthorized):
 
 
 class OAuthResponseProblem(OAuthProblem):
-    def __init__(self, token_response, **kwargs):
-        self.token_response = token_response
```

Comment:
what are the implications of removing this?

Reply:
It looks like nothing uses it. Everywhere that `OAuthResponseProblem` is raised, `token_response` is set to `None`.

If we remove it, we should also remove it from `docs/exceptions.rst`:

> This exception is thrown when there is a validation issue from your OAuth 2 Server. It contains a
> ``token_response`` property which contains the full http response from the OAuth 2 Server

If we decide to retain it, then we'll probably need to actually include the response here. But this is actually raised in places that have nothing to do with an oauth response... That's fishy. :(

`get_tokeninfo_func` calls `get_token_info_remote` to actually request an oauth token, so I would expect this to only be raised in that context, but those do not raise anything. I think we need a new exception named appropriately for how it is actually used. Any recommendations on what to name it?

Reply:
Some history here - every security handler used to raise OAuthResponseProblem in case of an error, so including it (or inheriting from it) was for backwards compatibility.
Hopefully we can drop that requirement at some point.

Reply:
Oh. Well, I just got rid of it everywhere where it was in use, and added it where it made sense.

Reply:
If we need to adjust the PR to split that out into another PR then fine. This PR wore me out, so if changes need to occur I'll wait a bit to make them.